lib/attributor/attribute.rb

```diff
@@ -137,8 +137,7 @@ def example_from_options(parent, context)
       when ::Regexp
         val.gen
       when ::Array
-        # TODO: handle arrays of non native types, i.e. arrays of regexps.... ?
-        val.pick
+        val
```

Comment:
yes, will fully remove the case. And we'll cut a new major version of the gem to make sure people don't stumble into the incompatibility change.

For somebody that wants to provide multiple examples, they can write a proc themselves that returns random samples if that's what they want to do.